content/blog/2023/fastapi-path-operations-for-django-developers.md

```diff
@@ -0,0 +1,258 @@
+---
+title: FastAPI Path Operations for Django Developers
+author: ed
+date: 2023-10-24
+tags:
+  - Article
+  - Python
+  - Django
+  - FastAPI
+image:
+  src: blog/2023/fast.jpg
+summary: |
+  FastAPI path operations are the equivalent to Django views. In this article we
+  explore the differences, advantages, and gotchas of using them from the
+  perspective of a Django developer.
+---
+
+You might have heard about FastAPI, a modern and fast web framework for building
+APIs with Python. You might be wondering how it compares to Django, the most
```

Comment:
Repeated use of "you might". Maybe combine these sentences, e.g:

"If you've heard about FastAPI, a modern ... with Python, you might be wondering how it compares..."